src/routes/(console)/project-[region]-[project]/databases/database-[database]/table-[table]/rows/columns/types/point.svelte

```
@@ -0,0 +1,56 @@
+    import { IconPlus } from '@appwrite.io/pink-icons-svelte';
+
+    interface Props {
+        label: string;
+        value: number[];
+        limited: boolean;
+        column: Models.ColumnPoint;
+    }
+
+    let { label, value, limited = false, column }: Props = $props();
+
+    let nullable = $state(false);
```

ESLint

[error] Prefer using writable $derived instead of $state and $effect

(svelte/prefer-writable-derived)